### ISSUE TITLE
Scheduling downtime gives stack trace

### ISSUE THREAD

**Issue by quentinsch** [Icinga/icingaweb2#3546]
When scheduling downtime for a host with all services checked it gives a stack trace in the UI. Although the downtime is scheduled properly.

Stack trace error:

![screen shot 2018-08-02 at 11 42 27](https://user-images.githubusercontent.com/7438854/43576440-aca12ae0-9649-11e8-9572-31780b861c7f.png)

Text of stack trace:
```
#0 /usr/share/icingaweb2/modules/monitoring/application/forms/Command/Object/ScheduleServiceDowntimeCommandForm.php(192): Icinga\Module\Monitoring\Command\Transport\CommandTransport->send(Object(Icinga\Module\Monitoring\Command\Object\ScheduleHostDowntimeCommand))
#1 /usr/share/icingaweb2/modules/monitoring/application/forms/Command/Object/ScheduleHostDowntimeCommandForm.php(113): Icinga\Module\Monitoring\Forms\Command\Object\ScheduleServiceDowntimeCommandForm->scheduleDowntime(Object(Icinga\Module\Monitoring\Command\Object\ScheduleHostDowntimeCommand), Object(Icinga\Web\Request))
#2 /usr/share/php/Icinga/Web/Form.php(1158): Icinga\Module\Monitoring\Forms\Command\Object\ScheduleHostDowntimeCommandForm->onSuccess()
#3 /usr/share/icingaweb2/modules/monitoring/library/Monitoring/Web/Controller/MonitoredObjectController.php(130): Icinga\Web\Form->handleRequest()
#4 /usr/share/icingaweb2/modules/monitoring/application/controllers/HostController.php(156): Icinga\Module\Monitoring\Web\Controller\MonitoredObjectController->handleCommandForm(Object(Icinga\Module\Monitoring\Forms\Command\Object\ScheduleHostDowntimeCommandForm))
#5 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action.php(507): Icinga\Module\Monitoring\Controllers\HostController->scheduleDowntimeAction()
#6 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch(String)
#7 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#8 /usr/share/php/Icinga/Application/Web.php(300): Zend_Controller_Front->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#9 /usr/share/php/Icinga/Application/webrouter.php(104): Icinga\Application\Web->dispatch()
#10 /usr/share/icingaweb2/public/index.php(4): require_once(String)
#11 {main}
```

## Expected Behavior
No error at all.

## Current Behavior
Produces stack trace, but executes downtime schedule as expected.

## Steps to Reproduce (for bugs)
1: Schedule downtime for a host with all services checked.

## Your Environment
* Icinga Web 2 version and modules (System - About): master (26-07-18)
* Version used (`icinga2 --version`): r2.9.1-1
* Operating System and version: Debian 8.11
* Enabled features (`icinga2 feature list`): api checker command graphite ido-mysql livestatus mainlog notification perfdata
* Config validation (`icinga2 daemon -C`): Validated OK

**Comment by dnsmichi** [comment]
Same as #3535 and patch in #3537

**Comment by lippserd** [comment]
Fixed with Web 2 2.6.1.
